### PR TITLE
Finished Student Lecture Ant Colony Optimization

### PR DESCRIPTION
**Pull request by DippyMcSkippy**
:mortar_board: 

### What

Finished Student Lecture Ant Colony Optimization

### Why

To merge with site ahead of lecture date

### Where

In the student lecture section

### CODE REVIEW

**Review by jameshughes89 — CHANGES_REQUESTED** [review]
Something is wrong. 
image

File: site/student-lectures/ant/ants.html
Comment (DippyMcSkippy) [inline site/student-lectures/ant/ants.html:1]: This file was refactored into Ant Colony Optimization.html

File: site/student-lectures/ant/base backyard.jpg
Comment (DippyMcSkippy) [inline site/student-lectures/ant/base backyard.jpg:1]: This was an extra version of a an image that already existed

File: site/index.rst
Comment (DippyMcSkippy) [inline site/index.rst:1]: This was part of the refactorization of ant/ants into Ant Colony Optimization/Ant Colony Optimization

**Comment by DippyMcSkippy** [comment]
I am not sure where else the problem could be I have built the site locally and it works fine

**Comment by jameshughes89** [comment]
You will need to fix the PR to not be rewriting the whole repo.

**Comment by jameshughes89** [comment]
> I am not sure where else the problem could be I have built the site locally and it works fine

I believe the problem is that you have build the website and saved the output to the repository, and committed those changes. 

I sent you an email with some advice.

**Comment by jameshughes89** [comment]
Still have at the .html file for the index and I believe the one for your lecture too

**Comment by DippyMcSkippy** [comment]
I removed those files this time

**Review by jameshughes89 — APPROVED** [review]
Ideally you have your py file within the src folder with unittests too, but this is fine for the lecture this week.

**Comment by DippyMcSkippy** [comment]
Thank You for your help

**Pull request merged**